### ISSUE TITLE
5hmc validation data

### ISSUE THREAD

**Issue by faulk-lab** [nanoporetech/dorado#167]
Are there any available datasets that show validation of 5hmC calls (with any model)? I have generated my own 0% and 100% 5mC DNA and can validate that myself, but for 5hmC that has been a challenge. I don't know how to create 5hmC in vitro at the bench, and zymo's 100% 5hmC 897 bp amplicon proved to be not a good substrate for this application.

All I need is some evidence of the accuracy of 5hmC calling. Anything out there that I am missing?

**Comment by faulk-lab** [comment]
Is there community interest in this? I have some genomic DNA now with validated 5mC and 5hmC levels. I could run it with Kit 14 on an R10.4.1 flow cell and validate the called percentages with the LC-MS percents. I'll do it only if there's interest, b/c I'm very busy.

**Comment by marcus1487** [comment]
We have presented 5hmC ground truth synthetic validation data at the last three major conferences (as well as for other modified bases). You can find the latest updates to the presented validation accuracy [in this video](https://www.youtube.com/watch?v=q8JCYAsuLUo) as well as a bit more background on the validation sample data type [in this video](https://youtu.be/01Q6_6SzLxo?si=jkUgTrcCPvFUEJPi). We have not yet publicly released this data, but have a plan to release synthetic raw data this year.

I would note that 100% modified strands are likely not the best data type for producing accuracy metrics. We do not target the modified base models to the highest range of modified base content (though we do attempt to include some high concentration modified base data into training where possible). We do this in order to maintain the highest accuracy possible in the more biologically relevant samples, generally containing 1-30% of bases modified. Thus you might expect accuracy metrics below those we report when using a 100% modified base sample. From the conference talks you can see that the validation strands we use for accuracy reporting are short (~100 base pairs) printed oligonucleotides with known modified base locations. 

I hope these resources help, but please do reach out if more details would be useful.

**Comment by vellamike** [comment]
Closing this issue due to inactivity. @faulk-lab if you need further clarification please request that I re-open.
